noxfile.py

```diff
@@ -22,7 +22,7 @@ def lint(session):
     session.run("pylint", "src")
 
 
-@session(python=["3.6", "3.7", "3.8", "3.9"], reuse_venv=True)
+@session(python=["3.8"], reuse_venv=True)
```

Comment:
why the change here?

Reply:
I don't see the point in running`coverage` in many versions, changed it to the oldest one supported.